### DISCUSSION TITLE
Mitigation of CVE-2022-40982 (Downfall/Inception) in OpenSSL

### DISCUSSION THREAD

**Discussion by Liu-ErMeng** [openssl/openssl#21761] [Q&A] [answered]
I have found the [CVE-2022-40982](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-40982), it's called  Intel GDS/DOWNFALL, AMD INCEPTION. 
linux kernel have merged some patch to mitigate this hardware vulnerability. Is there openssl have any plan to do some patches? 

reference:
https://nvd.nist.gov/vuln/detail/CVE-2022-40982
https://downfall.page/
https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=64094e7e3118aff4b0be8ff713c242303e139834

**Comment by hlandau** [comment, marked as answer]
Hello,

We have a page explaining the implications of Downfall and Inception here.

https://www.openssl.org/blog/blog/2023/08/15/downfall/

This is not a bug in OpenSSL, nor is it something we could mitigate. The mitigation needs to be applied to your system, not a single application or library such as OpenSSL. If you apply the mitigation advice given on the above page, you will be protected against these vulnerabilities, including when using OpenSSL.
